dcos/cluster.py

```diff
@@ -106,7 +106,7 @@ def setup_cluster_config(dcos_url, temp_path, stored_cert):
     try:
         # find cluster id
         cluster_url = dcos_url.rstrip('/') + '/metadata'
-        res = http.get(cluster_url, timeout=1)
+        res = http.get(cluster_url, timeout=http.FAIL_FAST_TIMEOUT)
```

Comment:
This changes the default timeout for this operation from 1 second to 3. Is that desired?

It also is not clear from the public documentation what will and what will not be affected by the user-set timeout configuration variable.